default_app/forkedProcess.js

```diff
@@ -0,0 +1,12 @@
+const os = require('os')
+const path = require('path')
+
+let productName = 'Child Product';
+let companyName = 'Child Company';
+let tmpPath = path.join(os.tmpdir(), productName + ' Crashes');
+
+process.startCrashReporter(productName, companyName, 'http://localhost:8080/uploadDump/childDump', tmpPath, {
```

Comment:
The signature here needs to match the signature in AtomBindings::StartCrashReporter.

If all the parameters were just strings, then yes, we could change the signature for the C++ method to be `AtomBindings::StartCrashReporter(const std::map<std::string, std::string> options)` 
But the last parameter is a key value pair in itself. I don't know how to support this scenario in C++.

What I have right now is exactly what is in https://github.com/electron/electron/blob/master/lib/common/api/crash-reporter.js#L65